iree/turbine/kernel/wave/runtime/setup.py

```diff
@@ -0,0 +1,57 @@
+from setuptools import setup, Extension
```

Comment:
Source files should have copyright headers (we can discuss shortening the headers, but we should generally be consistent within the repository).

Reply:
Sure I will put up a PR with the headers in this file.